lib/firefox/webdriver/firefoxDelegate.js

```diff
@@ -63,8 +64,12 @@ class FirefoxDelegate {
         ${chosenFeatures.length},${threadString},${chosenThreads.length});`;
       } else if (parameterLength === 5) {
         script = `Services.profiler.StartProfiler(${bufferSize},${interval},${featureString},${threadString});`;
+      } else if (parameterLength === 6) {
+        // XXX what about aFilters, aActiveBrowsingContextID and aDuration ?
```

Comment:
These are optional params according to https://searchfox.org/mozilla-central/rev/ea63a0888d406fae720cf24f4727d87569a8cab5/tools/profiler/gecko/nsIProfiler.idl#38

Reply:
Yeah I know. But what about them. Should we use them ? 
cc Florian

Reply:
I think aFilters here is ${threadString} and you'll want it.

You'll want to specify aActiveBrowsingContextID if you would like the profiler to know which tab you care about. This seems desirable for browsertime, but probably unrelated to what you are trying to do. https://hg.mozilla.org/mozilla-central/rev/59139e8c6636 has examples.

I think aDuration is irrelevant here.

Reply:
Thanks @fqueze - as this point browsertime is doing everything in the same tab afaik, so I guess this is irrelevant for now